e2e/core/global-setup.ts

```diff
@@ -15,7 +15,7 @@ async function globalSetup() {
   const token = Buffer.from(`${process.env.E2E_USER_ADMIN_USERNAME}:${process.env.E2E_USER_ADMIN_PASSWORD}`).toString(
     'base64',
   );
-  await requestContext.post(`${process.env.E2E_WS_BASE_URL}/rest/v1/session`, {
+  await requestContext.post(`${process.env.E2E_WS_BASE_URL}rest/v1/session`, {
```

Comment:
Can't we just remove the `/` from the env variable itself? I kind a prefer it the way it is cuz it shows that /rest is the path of the api. WDYT @jayasanka-sack?

Reply:
Yes, that would be clean. The reason to configure the env variable was the base URL of the UI and the API fixture was expecting the paths with a forward slash. However, I've noticed that some people are confused about needing to specify different hosts for WS and UI URLs, when in fact they should be the same. To simplify things, I will update the code to use a single environment variable, such as HOST, to specify the base URL for both UI and API requests. This should help to avoid any confusion and make it easier for everyone to use.